.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
```

Comment:
It would make sense to use just one CI, either Travis either Github Actions, not both. Do you prefer Github Actions?

Reply:
sorry my bad LOL I didn't see travis config file 🤦 😄 I'll remove this and use travis instead...